R/parsed.R

```diff
@@ -1,3 +1,15 @@
+#' Parse and pre-process character vector
+#'
+#' @description The function obtains detailed parse information for `text` via
+#'   [utils::getParseData] and does some minimal pre-processing by calling
+#'   [add_ws_to_parse_data].
+#' @inheritSection add_ws_to_parse_data Details
```

Comment:
I agree that small functions with well defined scope are preferable, but it can also add more overhead than benefit.